src/status.js

```diff
@@ -0,0 +1,14 @@
+function updateStatus(textElement, checkBox, task) {
+  textElement.classList.add('strike', 'text-muted');
+  const tick = document.createElement('i');
+  tick.classList.add('fa', 'fa-check', 'text-primary', 'me-1');
+  checkBox.parentNode.replaceChild(tick, checkBox);
+  task.completed = true;
+
+  const allTasks = JSON.parse(localStorage.getItem('tasks'));
+  const currentTask = allTasks.find((t) => t.index === task.index);
+  currentTask.completed = true;
+  localStorage.setItem('tasks', JSON.stringify(allTasks));
+}
+
+export default updateStatus;
```

Comment:
- [ ] For now I am only able to check the task to make it completed but I can't able to uncheck the completed task. Please make sure to work this in both ways so that if someone make the task complete by mistake should able to change the status. :slightly_smiling_face: